app/src/test/java/edu/uc/brown5a7/BottomsUp/DrinkDTOMockkUnitTest.kt

```diff
@@ -28,34 +27,21 @@ class DrinkDTOMockkUnitTest {
 
     var drinkService = mockk<DrinkService>()
 
-    @Test
-    fun searchForCoke_returnsCoke(){
-        givenAFeedOfMockkedDrinkDataAreAvailable()
-        whenSearchForCoke()
-        thenResultContainsCoke()
-        thenVerifiedCokeFunctionsEnvoked()
-    }
-
-    private fun thenVerifiedCokeFunctionsEnvoked() {
-        verify { drinkService.fetchDrinks("coke") }
-        verify(exactly = 0) {drinkService.fetchDrinks("aadskfl") }
-        confirmVerified(drinkService)
-    }
-
-    private fun givenAFeedOfMockkedDrinkDataAreAvailable() {
+    @Before
+    fun setup()
+    {
         mvm = MainViewModel()
         createMockkData()
     }
```

Comment:
This change was in the Sprint1 code review, but didn't quite get carried over. I see it reverted to the original implementation in commit 29a12af1f4c0f219d80b2afc1d718cc7188ea4aa, but I'm not 100% sure if that is intentional. For that reason, I have re-added back in the original changes from the Sprint 1 Code Review.